sockets/1.2/modules/ROOT/pages/index.adoc

```diff
@@ -1,92 +1,54 @@
-* Custom Class Loading Length Protocol +
-  A length protocol that uses a specific class loader to load objects from streams.
-* Safe Protocol +
-  A test protocol that precedes every message with a cookie, and should not be used in production environments.
-* Streaming Protocol +
-  Allows the socket's Send operation to return a message with the original `InputStream` as payload.
-* XML Message Protocol +
-  The `XmlMessageProtocol` is a protocol used to read streaming XML documents. The only requirement is that each document includes an XML declaration at the beginning of the document of the form "<?xml...". Make sure that the XML documents being streamed begin with an XML declaration when using this protocol. +
-  Data is read until a new document is found or until there is no more currently available data. For slower networks, `XmlMessageEofProtocol` might be more reliable. +
-  Also, because the default character encoding for the platform is used to decode the message bytes when looking for the XML declaration, some caution with message character encodings is warranted
-* XML Message EOF Protocol +
-  Extends `XmlMessageProtocol` to continue reading until either a new message or EOF is found.
-* Custom Protocol +
-  Define your own custom protocol by writing a class that extends `TcpProtocol`.
-
-These application level protocols are implemented on top of TCP. For these protocols to work properly, implement these in both client and server ends.
-
-By default, a socket implements Safe Protocol, but you can specify a different protocol in the `config` element. The following example, demonstrates how to configure the socket to use Direct Protocol:
-
-[source,xml,linenums]
-----
-	<sockets:listener-config name="Sockets_Listener_config" doc:name="Sockets Listener config" >
-		<sockets:tcp-listener-connection host="localhost" port="8082" >
-			<sockets:protocol>
-				<sockets:direct-protocol />
-			</sockets:protocol>
-		</sockets:tcp-listener-connection>
-	</sockets:listener-config>
-----
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to Anypoint Platform and Anypoint Studio.
+
+== Common Use Cases For the Connector
+
+Sockets Connector enables you to send or receive transport layer packages through both UDP and TCP sockets.
+
+For examples, see xref:sockets-connector-examples.adoc[Examples].
+
+
+== Audience
+
+* New user
++
+To create your Mule app, see:
+
+* xref:sockets-connector-studio.adoc[Studio Configuration]
+* xref:sockets-connector-config-topics.adoc[Additional Configuration Information]
+to help extend your understanding of the <connector-name> connector
+* xref:sockets-connector-examples.adoc[Examples], which provides one or more use cases for the connector.
++
+* Advanced user
++
+* Read the following:
```

Comment:
Nit pick: Instead of "Read the following", you could just suggest they refer to it: "Refer to:"